scripts/terminalserver/terminalserver.jl

```diff
@@ -18,6 +18,24 @@ end
 include("repl.jl")
 include("../debugger/debugger.jl")
 
+function getVariables()
+    M = Main
+    variables = []
+    for n in names(M)
+        !isdefined(M, n) && continue
+        x = getfield(M, n)
+        x isa Module && continue
+        x==Main.vscodedisplay && continue
+        n==Symbol("@run") && continue
+        n==Symbol("@enter") && continue
+        t = typeof(x)
+        value_as_string = Base.invokelatest(repr, x)
```

Comment:
guess we want to filter out gensymed variables here ? something like `startswith(value_as_string, '#') && continue`

Reply:
Yes, agreed, I made a change for that.